pkgs/development/libraries/flann/default.nix

```
@@ -1,24 +1,58 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, unzip, cmake, python }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, unzip
+, cmake
+, python ? null
```

```suggestion
, python
```
can this be python3?

---

Yeah, I think that'll work fine. I wasn't sure what the policy on using `python3` explicitly— a lot of packages seem to just use `python`, eg:

https://github.com/NixOS/nixpkgs/blob/nixos-21.05/pkgs/development/libraries/boost/generic.nix#L1

---

Boost is not a good example here because the python there gets most of the time overwritten with something else. Outside of pytonPackages python3 should always be used.